scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -556,8 +556,12 @@ class Router(formatOps: FormatOps) {
         Seq(
           Split(modification, 0)
         )
-      // Defn.{Object, Class, Trait}
-      case FormatToken(_: T.KwObject | _: T.KwClass | _: T.KwTrait, r, _) =>
+      // Defn.{Object, Class, Trait, Enum}
```

Comment:
forgot to ask. should we assume that if the dialect doesn't support enum, KwEnun would not be parsed?

Reply:
Yes, it will not be parsed in that case. I will be a normal Ident.

Reply:
That's a good question. As I play with the dialect-specific features right know I can see that they parse even if they're not enabled in the dialect. I assume that it should be parsed then?